src/components/marker-chart/Canvas.js

```diff
@@ -587,11 +588,32 @@ class MarkerChartCanvasImpl extends React.PureComponent<Props> {
       }
 
       const y = rowIndex * rowHeight - viewportTop;
+
+      let countString = '';
+      if (drawMarkerCount) {
+        let count = markerTiming.length;
+        for (
+          let row = rowIndex + 1;
+          row < markerTimingAndBuckets.length;
+          ++row
+        ) {
+          if (
+            typeof markerTimingAndBuckets[row] === 'string' ||
+            markerTimingAndBuckets[row].name !== name
+          ) {
+            break;
+          }
+          count += markerTimingAndBuckets[row].length;
+        }
```

Comment:
Is it worth extracting the algorithm to compute the marker count to a separate method?
`countMarkerInBucketStartingAtRow(rowIndex)`